inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp

```diff
@@ -17,7 +17,10 @@
 #include "kernel_selector_utils.h"
 #include <string>
 #include <functional>
+#include <cmath>
+
 #define CEIL_DIV(A, B) ((A + B - 1)/(B))
+#define VECTORWIDTH 8
```

Comment:
please remove this, this is currently define din param.h

Reply:
(of course that will be moved to permute_kernel_tile_8x8.h...

Reply:
Okay, I removed VECTOWIDTH macro